guides/common/modules/proc_setting-an-alternative-directory-for-remote-execution-jobs-in-push-mode.adoc

```diff
@@ -0,0 +1,36 @@
+[id="Setting_an_Alternative_Directory_for_Remote_Execution_Jobs_in_Push_Mode_{context}"]
```

Comment:
The anchor ID format in the other file is more readable and aligns with modular doc requirements. I recommend you change this to `[id="setting-an-alternative-directory-for-remote-execution-jobs-in-push-mode_{context}"]`

Reply:
Applied.